policyengine_uk/variables/gov/hmrc/tax_free_childcare/conditions/tax_free_childcare_work_condition.py

```diff
@@ -0,0 +1,57 @@
+from policyengine_uk.model_api import *
+
+
+class tax_free_childcare_work_condition(Variable):
+    value_type = bool
+    entity = Person
+    label = "Work conditions for tax-free childcare"
+    documentation = "Whether the person/couple meets work requirements for tax-free childcare"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        """Calculate if person meets work conditions for:
+        - Single working adult
+        - Couple where either both work or one works and other has disability/incapacity
+        """
+        benunit = person.benunit
+        is_adult = person("is_adult", period)
+
+        # Basic work status
+        in_work = person("in_work", period).astype(bool)
+
+        # Get disability/incapacity conditions like we did in childcare age eligibility
+        gc = parameters(period).gov.dwp.pension_credit.guarantee_credit
+        standard_disability_benefits = gc.child.disability.eligibility
+        severe_disability_benefits = gc.child.disability.severe.eligibility
+
+        receive_disability_program = (
+            (add(person, period, standard_disability_benefits) > 0)
+            | (add(person, period, severe_disability_benefits) > 0)
+        ).astype(bool)
```

Comment:
Remove all `.astype(bool)`s - this should almost always be unnecessary (always unnecessary in instances in this PR)